colossalai/utils/checkpointing.py

```diff
@@ -124,35 +125,27 @@ def partition_pipeline_parallel_state_dict(model, state_dict):
 
 
 def gather_pipeline_parallel_state_dict(state_dict):
-    gathered_states = (
-        [None for _ in range(gpc.get_world_size(ParallelMode.PIPELINE))]
-        if gpc.get_local_rank(ParallelMode.PIPELINE) == 0
-        else None
-    )
+    gathered_states = ([None for _ in range(gpc.get_world_size(ParallelMode.PIPELINE))]
+                       if gpc.get_local_rank(ParallelMode.PIPELINE) == 0 else None)
     dist.gather_object(
         state_dict,
         gathered_states,
         dst=gpc.get_ranks_in_group(ParallelMode.PIPELINE)[0],
         group=gpc.get_cpu_group(ParallelMode.PIPELINE),
     )
 
-    state_dict = (
-        OrderedDict(chain.from_iterable(state.items() for state in gathered_states))
-        if gpc.get_local_rank(ParallelMode.PIPELINE) == 0
-        else OrderedDict()
-    )
+    state_dict = (OrderedDict(chain.from_iterable(state.items() for state in gathered_states))
+                  if gpc.get_local_rank(ParallelMode.PIPELINE) == 0 else OrderedDict())
 
     return state_dict
 
 
-def save_checkpoint(
-    file,
-    epoch: int,
-    model: torch.nn.Module,
-    optimizer: torch.optim.Optimizer = None,
-    lr_scheduler: torch.optim.lr_scheduler._LRScheduler = None,
-    **kwargs
-):
+def save_checkpoint(file,
+                    epoch: int,
+                    model: torch.nn.Module,
+                    optimizer: torch.optim.Optimizer = None,
+                    lr_scheduler: torch.optim.lr_scheduler._LRScheduler = None,
```

Comment:
Optional

Reply:
@kurisusnowdeng I notice that you comment codes abourt save/load optimizer and lr scheduler. Why?